bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx

```diff
@@ -369,49 +369,49 @@ class PresentationUploader extends Component {
 
       const modPresentation = presentation;
       if (currentPropPres?.current !== prevPropPres?.current) {
-        modPresentation.current = currentPropPres.current;
+        modPresentation.current = currentPropPres?.current;
```

Comment:
Instead of these ~20 checks for whether `currentPropPres` is defined, can't we put something like this in the beginning:
`if (!currentPropPres) return;`

Aside from it -- in all 8 instances where you add a `?` check, we already check this in the outer `if`... I am not sure how this is helping with the issue. I might be missing something though...

Reply:
The if conditions don’t actually check whether currentPropPres exists; they just safely reference its properties with optional chaining so it won’t error out if currentPropPres is undefined.

Reply:
I am testing this and indeed I have not hit the issue with your code. I am still struggling to see why you'd need to use optional chaining inside the body of the if statement if you had the same optional chaining in the condition? My understanding is that the call would be short circuited in the condition (where we already had `?`) and would not even enter the body.

I think I will proceed with merging this as I don't see real drawbacks of it.

And what did you think of exiting early (way before the checks) if `currentPropPres` is Falsy?